### PR TITLE
chore: create security.txt

### PR DESCRIPTION
**Pull request by daallgeier**
Adding a security.txt file enables security researchers to quickly and easily see where they can submit security issues and know that they are being taken serious.  From the proposal website: 

> "When security risks in web services are discovered by independent security researchers who understand the severity of the risk, they often lack the channels to disclose them properly. As a result, security issues may be left unreported. security.txt defines a standard to help organizations define the process for security researchers to disclose security vulnerabilities securely.”

See also https://securitytxt.org

### CODE REVIEW

**Comment by Mythie** [comment]
Hey there, it looks like you haven't accepted our contributor license agreement yet. In order for us to accept your pull request we ask that you please fill out the CLA:

https://documen.so/cla

**Comment by ElTimuro** [comment]
- @daallgeier makes sense
- I like adding a minimal version first
- Should this contain also an email for critical vulnerabilities to ensure time to react? Reporting everything publicly looks not ideal.

**Comment by daallgeier** [comment]
- great
- it's supposed to be minimal. It's not a policy. 
- adding an email address like security@ helps for sure. Would be awesome if GH would have a way to create issues marked as confidential that are only visible to the reporter, any cc'd, and the gh team but for now it shall be an email address. 

Do that by adding another line with contact and either just the address or mailto:security@...

**Comment by Mythie** [comment]
Should probably be under the public directory in both the marketing and webapp, we can use a script to copy paste it as we've done for pdfjs.

**Comment by daallgeier** [comment]
I guess that makes sense.  I am not sure about (your) mono repo architecture(s).

**Comment by Mythie** [comment]
All sorted! Thanks Tangerine 🍊

**Review by Mythie — DISMISSED** [review]

File: .well-known/security.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# General Issues
+Contact: https://github.com/documenso/documenso/issues/new?assignees=&labels=bug&projects=&template=bug-report.yml
+# Report critical issues privately, to let us take appropriate action before publishing
```
Comment (catalinpit) [inline .well-known/security.txt:3]: This should be updated to "Report critical issues privately to let us take appropriate action before publishing.", I believe.

Also, for better readability, maybe add space before this paragraph? Just a suggestion

File: apps/web/public/.well-known/security.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# General Issues
+Contact: https://github.com/documenso/documenso/issues/new?assignees=&labels=bug&projects=&template=bug-report.yml
+# Report critical issues privately, to let us take appropriate action before publishing
```
Comment (catalinpit) [inline apps/web/public/.well-known/security.txt:3]: Same as in the other `security` file

File: scripts/copy-wellknown.cjs [hunk 3]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env node
+
+/* eslint-disable @typescript-eslint/no-var-requires */
+const path = require('path');
+const fs = require('fs');
+
+const wellKnownPath = path.join(__dirname, '../.well-known');
+
+console.log('Copying .well-known/ contents to apps');
```
Comment (catalinpit) [inline scripts/copy-wellknown.cjs:9]: is this console log necessary?

File: scripts/copy-wellknown.cjs [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+fs.cpSync(wellKnownPath, path.join(__dirname, '../apps/web/public/.well-known'), {
```
Comment (catalinpit) [inline scripts/copy-wellknown.cjs:10]: `cpSync` is in Stage 1 and it says the following on their page *Use of the feature is not recommended in production environments.*

Should we go ahead and use it?

File: apps/marketing/public/.well-known/security.txt [hunk 5]
```diff
@@ -0,0 +1,6 @@
+# General Issues
+Contact: https://github.com/documenso/documenso/issues/new?assignees=&labels=bug&projects=&template=bug-report.yml
+# Report critical issues privately, to let us take appropriate action before publishing
```
Comment (catalinpit) [inline apps/marketing/public/.well-known/security.txt:3]: Same as in the other `security` file.

**Comment by ElTimuro** [comment]
- Created security@documens.com

File: .well-known/security.txt (hunk 1 above)
Comment (ElTimuro) [reply .well-known/security.txt:3]: done. feel free to suggest edit like these directly, i would just have added them

File: scripts/copy-wellknown.cjs (hunk 4 above)
Comment (ElTimuro) [reply scripts/copy-wellknown.cjs:10]: @Mythie unsure about this. other than that, this is good to go IMO

Comment (Mythie) [reply scripts/copy-wellknown.cjs:10]: It's a script rather than something in production it's fine.

File: scripts/copy-wellknown.cjs (hunk 3 above)
Comment (Mythie) [reply scripts/copy-wellknown.cjs:9]: It'll double log since we also echo in the pre-commit but I'd say this is also fine 😄

**Review by Mythie — APPROVED** [review]

**Pull request merged**